.github/workflows/after-merge.yml

```diff
@@ -0,0 +1,157 @@
+# Run after merge to trunk
```

Comment:
I think this only works as long as *Require branches to be up to date before merging* is enabled in the branch protection rules. (Without that someone could merge to master a state that wasn't tested in the PR.)

Would be good to note that requirement.